ChangeLog

```diff
@@ -1,3 +1,10 @@
+maid (0.9.0) unstable; urgency=low
```

Comment:
Do you have any rule of thumb to choose between `stable` and `unstable`?

Reply:
My process was always to do at least one unstable release before a stable release so that people could give feedback.  There is a list of users who have shared rules in [the wiki](https://github.com/benjaminoakes/maid/wiki).  Sometimes those users would help QA the releases and sometimes found issues, as I recall.

Reply:
You could look at what I did in old commits (it's been a while!), but I recall that once the `VERSION` became a stable version (i.e., change `0.9.0.alpha.1` to `0.9.0`), I would change the `ChangeLog` to read `stable` for that version.